test/test_dynamic_shapes.py

```diff
@@ -178,6 +178,37 @@ def test_expand_symint_correctness(self):
     self.assertEqual(t3.shape[0], 2)
     self.assertEqual(expand_out_aten.cpu(), expand_out_xla.cpu())
 
+  def test_topk_symint(self):
+    dev = xm.xla_device()
+    t1 = torch.zeros([5, 2], device=dev)
+    t1[3][0] = 1
+    t1[3][1] = 1
+    t2 = torch.nonzero(t1)
+    t3 = torch.zeros([10, 2], device=dev)
+    values, indices = torch.topk(t3, t2.shape[0], dim=0)
+    self.assertIsInstance(t2.shape[0], torch.SymInt)
+    self.assertIsInstance(values.shape[0], torch.SymInt)
+    self.assertIsInstance(indices.shape[0], torch.SymInt)
+    self.assertEqual(str(t2.shape[0]), '<=10')
+    self.assertEqual(str(values.shape[0]), '<=10')
+    self.assertEqual(str(indices.shape[0]), '<=10')
+    self.assertEqual(t3.shape[1], 2)
+    self.assertEqual(values.shape[1], 2)
+    self.assertEqual(indices.shape[1], 2)
+
+    def test_fn(*tensors):
+      torch.manual_seed(0)
+      x = torch.rand(10, 10).to(tensors[0].device)
+      results = []
+      for tensor in tensors:
+        k = torch.nonzero(tensor).shape[0]
+        for dim in range(2):
+          results += list(torch.topk(x, k=k, dim=dim))
+      return results
+
+    self.runAtenTest([torch.randint(0, 2, size=(10,)) for _ in range(5)],
```

Comment:
this is to test the correctness of the `topk`?

Reply:
That's right. The `runAtenTest` function will run and compare the results with native pytorch.